app/views/ProfileView/index.tsx

```diff
@@ -295,6 +295,8 @@ class ProfileView extends React.Component<IProfileViewProps, IProfileViewState>
 
 			if (result) {
 				logEvent(events.PROFILE_SAVE_CHANGES);
+				params.name = params.realname;
+				delete params.realname;
```

Comment:
Is it really necessary to delete this field?

Reply:
I didn't delete it here and "nothing" happened.
What is the specific reason for deleting this field?

Reply:
I deleted it just to don't pass the `realname` param that will be a "new" param into redux and will be duplicated as the name